src/components/cart/CartSidebar/CartSidebar.tsx

```diff
@@ -19,21 +19,23 @@ import EmptyCart from '../EmptyCart'
 
 import './cart-sidebar.scss'
 
+type CB = () => unknown
```

Comment:
Avoid using abbreviations to improve readability :)

```suggestion
type callBack = () => unknown
```

Reply:
IMO, since this type is being used at many places, can create a type and reuse it, or can just inline `() => void | undefined`